src/models/service-params.ts

```diff
@@ -0,0 +1,12 @@
+import FSService from "../services/fs-service"
+import { GitService } from "../services/git-service"
+
+export default interface ServiceParams {
+  name: string
+  parentDirectory: string
+}
+
+export interface ServiceTools {
```

Comment:
This is not needed, make dependencies explicit in services

Reply:
removed